### PR TITLE
Graphics for existing playbooks

### PR DESCRIPTION
**Pull request by joshmoore**
Simple graphics based on the original templates provided by Liza
to get across what the highlights (nodes, services) of each playbook
are. Future versions may want to add groups, variables, etc.

cc: @kennethgillen

### CODE REVIEW

File: one-node/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+dependency:
+  name: galaxy
+  requirements_file: ../requirements.yml
+
+driver:
+  name: docker
+
+docker:
+  containers:
+  - name: omero-all
```
Comment (sbesson) [inline one-node/molecule.yml]: [Travis CI](https://travis-ci.org/ome/ansible-examples-omero/jobs/458473781) is failing with

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/bin/molecule", line 11, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/cli.py", line 41, in main
    cli(obj={})
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/command/test.py", line 96, in test
    util.sysexit(t.execute()[0])
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/command/test.py", line 44, in execute
    status, output = c.execute(exit=False)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/command/create.py", line 42, in execute
    self.molecule.driver.up(no_provision=True)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/driver/dockerdriver.py", line 133, in up
    self._build_ansible_compatible_image(container)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/molecule/driver/dockerdriver.py", line 260, in _build_ansible_compatible_image
    'image of {}:{} ...').format(container['image'],
KeyError: 'image'
```

Which base image should be used here?

Comment (joshmoore) [reply one-node/molecule.yml]: Uh, I assume this is a change in molecule, no?

**Comment by sbesson** [comment]
I pushed a commit to fix the Docker startup issue. Now Travis is failing with `fatal: [omero-all]: FAILED! => {"changed": false, "failed": true, "msg": "Unable to start service omero-server: Failed to start omero-server.service: Unit not found.\n"}` - see https://travis-ci.org/ome/ansible-examples-omero/jobs/485422414.

Looking at the content of the service file a few lines above in the log, could it be related to the fact `User=omero-server` has not been created i.e. is `role: omero-user` mandatory in the one-node playbook?

**Comment by joshmoore** [comment]
_grrr_ Still struggling with this.

 1. Added ice_python_wheel to go from 25 min to 15
 2. First pinned ansible and molecule to try to fix one problem
 3. Now just tried to pin testinfra to 1.6.3 to fix

```
INTERNALERROR>     session.config.warn('C1', msg)
INTERNALERROR> AttributeError: 'Config' object has no attribute 'warn'
```

**Comment by joshmoore** [comment]
> could it be related to the fact 

Think it was more related to the network setup. Fix pushed.

**Comment by joshmoore** [comment]
Based on my experience with f92f6c57f0931cbf6fce5f537425f8117813a042 I'd suggest we move this to the base images (omero-server-docker and omero-web-docker). cc @manics

**Comment by sbesson** [comment]
I assume the release of https://github.com/openmicroscopy/omero-web-docker/pull/26 will implement https://github.com/ome/ansible-examples-omero/pull/12#issuecomment-458608122 for the `omero-web` Docker image

**Comment by joshmoore** [comment]
> I assume the release of

Yup. I'll remove those here once that's released..

**Comment by joshmoore** [comment]
Green! 🎉

**Comment by sbesson** [comment]
:+1: Merging and I will start pointing people at this repository for minimal examples including the simple one-node setup.

**Pull request merged**